addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -93,6 +93,81 @@
     </record>
 
     <record id="hr_leave_view_kanban" model="ir.ui.view">
+        <field name="name">hr.leave.view.kanban</field>
```

Comment:
This view and `hr_leave_view_kanban_approve_department` seem quite similar except 2-3 differences, wouldn't it be better to have one view inherit from the other and only change what's needed? :-)